services/web/src/components/Link/JumpLink.js

```diff
@@ -39,7 +40,14 @@ class JumpLink extends React.Component {
     const { to, className = '', ...rest } = this.props;
     const classes = className.split(' ');
     classes.push('jump-link');
-    return <a href={`#${to}`} className={classes.join(' ')} {...rest} />;
+    return (
+      <Anchor
```

Comment:
I'm still a bit confused about what this component does or why it's necessary. These link components are meant to be unstyled anyway. It should be OK to use naked HTML elements when necessary.

Reply:
We covered this this is mantine's way of dealing with. connected to theming.